libqtile/scripts/migrate.py

```diff
@@ -193,8 +193,8 @@ def add_subcommand(subparsers, parents):
         help="Use the specified configuration file (migrates every .py file in this directory)",
     )
     parser.add_argument(
-        "--interactive",
-        action="store_true",
-        help="Interactively apply diff (similar to git add -p)",
+        "--non-interactive",
```

Comment:
What do you think of --yes or --quiet?

Reply:
`--yes` is ok, I can switch it to that. I think `--quiet` is kind of ambiguous.

Reply:
Yes please ❤️